wikipedia/track.py

```diff
@@ -266,6 +268,8 @@ def params(self):
                 query_body = {"bool": {"should": [{"match": {"title": query}}, {"match": {"content": query}}]}}
             elif self._query_type == "term":
                 query_body = {"bool": {"should": [{"term": {"title": query}}, {"term": {"content": query}}]}}
+            elif self._query_type == "match_phrase":
```

Comment:
Now that I see it - I wonder why we're doing match over content and not on content and title as the other queries 🤔 .

We're comparing apples to oranges a bit, as ESQL queries both. I think we should correct that - can be in a separate PR.

WDYT?

Reply:
Good point. I'll update for this PR.